src/Symfony/Component/Console/Tests/Command/DumpCompletionCommandTest.php

```diff
@@ -13,6 +13,10 @@ class DumpCompletionCommandTest extends TestCase
      */
     public function testComplete(array $input, array $expectedSuggestions)
     {
+        if (!class_exists(CommandCompletionTester::class)) {
+            $this->markTestSkipped('Test command completion requires symfony/console 5.4+.');
+        }
```

Comment:
This test case is part of the `symfony/console` package. We should not need to check if a class from the same package is present.